client/payment-details/timeline/map-events.js

```diff
@@ -321,6 +363,7 @@ const mapEventToTimelineItems = ( event ) => {
 					[
 						composeFXString( event ),
 						feeString,
+						...feeBreakup( event ),
```

Comment:
Is the spread operator `...` necessary here? I removed it and it's still working very well.

Reply:
And if `feeBreakup` were to return a React element (`<ul>`), it would even break.

Reply:
Fixed in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/25fc362ff7b03b354fab7b7117e6aeb4ef15ccce